manifests/eap/module.pp

```diff
@@ -195,24 +195,6 @@
         require => File[$directory],
     }
 
-    file { '/usr/share/jbossas/modules/com/redhat/lightblue/main/appconfig.properties':
-        mode    => '0644',
-        owner   => 'jboss',
-        group   => 'jboss',
-        content => template('lightblue/properties/appconfig.properties.erb'),
-        notify  => Service['jbossas'],
-        require => File[$directory],
-    }
-
-    file { '/usr/share/jbossas/modules/com/redhat/lightblue/main/lightblue-client.properties':
-        mode    => '0644',
-        owner   => 'jboss',
-        group   => 'jboss',
-        content => template('lightblue/properties/appconfig.properties.erb'),
-        notify  => Service['jbossas'],
-        require => File[$directory],
-    }
-
```

Comment:
Should we puppetize cleanup (removal) of the appconfig.properties and lightblue-client.properties files?  Else they hang out on systems forever.